metricbeat/module/kubernetes/pod/pod_test.go

```diff
@@ -44,12 +43,18 @@ func TestEventMapping(t *testing.T) {
 	body, err := ioutil.ReadAll(f)
 	assert.NoError(t, err, "cannot read test file "+testFile)
 
-	cache := util.NewPerfMetricsCache(120 * time.Second)
-	cache.NodeCoresAllocatable.Set("gke-beats-default-pool-a5b33e2e-hdww", 2)
-	cache.NodeMemAllocatable.Set("gke-beats-default-pool-a5b33e2e-hdww", 146227200)
-	cache.ContainerMemLimit.Set(util.ContainerUID("default", "nginx-deployment-2303442956-pcqfc", "nginx"), 14622720)
+	metricsRepo := util.NewMetricsRepo()
 
-	events, err := eventMapping(body, cache, logger)
+	nodeUID := "gke-beats-default-pool-a5b33e2e-hdww"
+	nodeMetricsRepoId := util.GetMetricsRepoId(util.NodeMetricSource, nodeUID)
+	metricsRepo.Set(nodeMetricsRepoId, util.NodeCoresAllocatableMetric, 2)
```

Comment:
wdyt about adding a test with the default value `0.0` here and for the scenario when we have multiple containers in the pod?